src/Apps/Order/Routes/Respond/index.tsx

```diff
@@ -70,6 +70,9 @@ export class RespondRoute extends Component<RespondProps, RespondState> {
         case "ACCEPT":
           this.props.router.push(`/orders/${this.props.order.id}/accept`)
           break
+        case "DECLINE":
+          this.props.router.push(`/orders/${this.props.order.id}/review/reject`)
```

Comment:
Note: we're going to go with `/decline` instead here. We'll need to update the `routes.tsx` file too.